lib/logstash/util/zeromq.rb

```diff
@@ -21,8 +21,8 @@ def setup(socket, address)
     @logger.info("0mq: #{server? ? 'connected' : 'bound'}", :address => address)
   end
 
-  def error_check(rc, doing)
-    unless ZMQ::Util.resultcode_ok?(rc)
+  def error_check(rc, doing, eagain_not_error = false)
```

Comment:
Because function signature of error_check is changed and util/zeromq.rb is present in zeromq input and output plugin, this PR has a dependecy to logstash-plugins/logstash-input-zeromq/pull/18.